client/blocks/login/error-notice.jsx

```diff
@@ -61,6 +61,11 @@ class ErrorNotice extends Component {
 			return null;
 		}
 
+		// The existing_wpcom_user error is handled inline in LoginForm
```

Comment:
I find that comment helpful. It tells us to look into `LoginForm` to see where this error is displayer. If we look at it we can see that this message is triggered here https://github.com/Automattic/wp-calypso/pull/16447/files#diff-29356c8042a170020e49bae30afff806R139 when the `SocialLoginForm` component calls `linkSocialUser()` https://github.com/Automattic/wp-calypso/pull/16447/files#diff-03f7d5aab5df5cd3f255095d092ff252R64